Cabal/Distribution/Simple/Utils.hs

```diff
@@ -823,7 +841,15 @@ rawSystemStdInOut verbosity path args mcwd menv input _ = withFrozenCallStack $
   printRawCommandAndArgs verbosity path args
 
   Exception.bracket
-     (runInteractiveProcess path args mcwd menv)
+     (do (Just inh,Just outh, Just errh,pid) <-
+           createProcess $ enableProcessJobs
```

Comment:
define inline `runInteractiveProcess`, which would be copy of what in `process` with redefined `createProcess`. Add the comment explaining why it's redefined.